scrapy_autoextract/errors.py

```diff
@@ -0,0 +1,27 @@
+from autoextract.aio.errors import DomainOccupied
+
+
+class QueryError(Exception):
+
+    def __init__(self, query: dict, message: str):
+        self.query = query
+        self.message = message
+
+    def __str__(self):
+        return f"QueryError: message='{self.message}, query={self.query}'"
+
+
+def summarize_exception(exc):
+    """
+    Provides a text that represents the exception. To be used in stats, so
+    produced text shouldn't be too diverse.
+    """
+    if isinstance(exc, QueryError):
+        msg = exc.message
+        if DomainOccupied.from_message(msg):
+            # Removes variability from domain occupy messages
+            msg = "domain occupied"
```

Comment:
I haven't tried it (maybe it is too noisy), but won't it be interesting to know which domains are causing domain occupied errors?